deleteAndLoadSnapsets.sh

```diff
@@ -31,7 +32,24 @@ if [ -z "$COUCHDB_URL" ]; then
 fi
 
 log "Starting"
+log "Clear index: $CLEAR_INDEX"
+log "Static: $STATIC_DATA_DIR"
+log "Build: $BUILD_DATA_DIR"
+log "Node path: $NODE_PATH"
+log "Working directory: `pwd`"
 
+# Set up universal
+git clone https://github.com/GPII/universal.git
```

Comment:
Thanks for making this change by dynamically pulling universal repo at every run.

Reply:
Thanks @cindyli.

You're right, the httpTests.js was just an investigative script to see how the http module worked, especially with CouchDB urls.  I've removed it.

I've also updated the README